media-proxy/src/proxy_context.cc

```diff
@@ -787,23 +787,59 @@ int ProxyContext::TxStart(const TxControlRequest* request)
         return -1;
     }
 
-    st_ctx = new (mtl_session_context_t);
+    st_ctx = new (dp_session_context_t);
     st_ctx->id = incrementMSessionCount();
     st_ctx->type = TX;
     st_ctx->tx_session = tx_ctx;
-    mStCtx.push_back(st_ctx);
+    mDpCtx.push_back(st_ctx);
 
     /* TODO: to be removed later. */
     mTxCtx.push_back(tx_ctx);
 
     return (st_ctx->id);
 }
 
-int ProxyContext::RxStart(const mcm_conn_param* request)
+int ProxyContext::RxStart_rdma(const mcm_conn_param* request) {
+    rdma_s_ops_t opts = { 0 };
+    dp_session_context_t* dp_ctx = NULL;
+    memif_ops_t memif_ops = { 0 };
+    rx_rdma_session_context_t* rx_ctx = NULL;
+    int ret;
```

Comment:
Reorder to look better. Add an empty line after variable declarations. All other similar places in this PR are subject to update.
```suggestion
    rx_rdma_session_context_t* rx_ctx = NULL;
    dp_session_context_t* dp_ctx = NULL;
    memif_ops_t memif_ops = { 0 };
    rdma_s_ops_t opts = { 0 };
    int ret;

```

Reply:
done in most of the places